src/main/java/io/zksync/signer/ZkSigner.java

```diff
@@ -20,16 +24,17 @@
 
 public class ZkSigner {
 
-    private final ZksCrypto crypto;
+    private static final ZksCrypto crypto = ZksCrypto.load();
+    
+    public static final String MESSAGE = "Access zkSync account.\n\nOnly sign this message for a trusted client!";
 
     private final ZksPrivateKey privateKey;
 
     private final ZksPackedPublicKey publicKey;
 
     private final String publicKeyHash;
 
-    private ZkSigner(ZksPrivateKey privateKey, ZksCrypto crypto) {
-        this.crypto = crypto;
+    private ZkSigner(ZksPrivateKey privateKey) {
         this.privateKey = privateKey;
 
         System.out.println("Private key: " + Numeric.toHexString(privateKey.getData()));
```

Comment:
Is that intended to print the private key?

Reply:
Yikes.